common/compositor/nativesurface.h

```diff
@@ -68,7 +68,7 @@ class NativeSurface {
   void SetInUse(bool inuse);
 
   bool InUse() const {
-    return ref_count_ > 1 || in_flight_;
+    return in_use_;
```

Comment:
We will need a spinlock here. Since InUse will be queried in application thread and SetinUse can be called from app thread and kmsfencehandler thread